Packs/GZipFile/Scripts/GZipFile/GZipFile.py

```diff
@@ -0,0 +1,82 @@
+import gzip
+import re
+import shutil
+from os.path import isfile
+
+import demistomock as demisto  # noqa: F401
+import pyminizip
+from CommonServerPython import *  # noqa: F401
+
+ESCAPE_CHARACTERS = r'[/\<>"|?*]'
+
+
+def escape_illegal_characters_in_file_name(file_name: str) -> str:
+    if file_name:
+        file_name = re.sub(ESCAPE_CHARACTERS, '-', file_name)
+        file_name = re.sub(r'-+', '-', file_name)  # prevent more than one consecutive dash in the file name
+
+    return file_name
+
+
+def main():
+    try:
+        args = demisto.args()
+        fileEntryID = args.get('entryID')
```

Comment:
please extract this to functions

Reply:
You mean to create a separate function for this?

Reply:
Yes, We prefer having a function to which receives the arguments, and return a `CommandResult` object.